pkgs/by-name/vu/vulnix/package.nix

```diff
@@ -6,22 +6,17 @@
   ronn,
 }:
 
-python3Packages.buildPythonApplication {
+python3Packages.buildPythonApplication rec {
   pname = "vulnix";
-  version = "1.10.2";
+  version = "1.11.0";
 
   src = fetchFromGitHub {
     owner = "nix-community";
     repo = "vulnix";
-    rev = "9abfc80da0b4135e982332e448a3969f3b28785b";
-    hash = "sha256-gSgAGN7LlciW4uY3VS49CbZ9WuRUcduJ5V7JesA8OVo=";
+    tag = version;
+    hash = "sha256-bQjmAmTRP/ce25hSP1nTtuDmUtk46DxkKWtylJRoj3s=";
   };
 
```

Comment:
```suggestion
__darwinAllowLocalNetworking = true;
```

Reply:
Thanks, Fixed in 7115a2b73d4e914496f426785c9b77be1961ca6c